Packs/IntegrationsAndIncidentsHealthCheck/Scripts/CopyLinkedAnalystNotes/CopyLinkedAnalystNotes.py

```diff
@@ -56,6 +56,11 @@ def main():
     if linked_incident:
         main_integration_grid, main_incident_grid = create_grids(custom_fields, linked_incident)
 
+        all_rows_have_date = all('creationdate' in row for row in main_incident_grid)
+        if all_rows_have_date:
+            main_incident_grid.sort(key=lambda incident_dict: incident_dict['creationdate'], reverse=True)
+            # sort incident grid by creation date
```

Comment:
```suggestion
      # sort incident grid by creation date
      main_incident_grid.sort(key=lambda incident_dict: incident_dict.get('creationdate'), reverse=True)
```

This would work?
If not, maybe adding a default value will work

Reply:
Seemed like it works, thanks.
I changed